_posts/pl/2023-11-18-czy-wiesz-czym-jest-opentelemetry-i-jak-go-stosować-w-praktyce.md

```diff
@@ -0,0 +1,136 @@
+---
+layout:    post
+title:     "Czy wiesz, czym jest OpenTelemetry i jak go stosować w praktyce?"
+date:      2023-11-18T08:00:00+01:00
+published: true
+didyouknow: false
+lang: pl
+author: fphilavong
+image: /assets/img/posts/2023-11-18-czy-wiesz-czym-jest-opentelemetry-i-jak-go-stosować-w-praktyce/otel.png
+tags:
+- OpenTelemetry
+- tracing
+- span
+- trace
+- Jaeger
+---
+
+## Słowem wstępu
+
+OpenTelemetry to zbiór SDK i interfejsów API, który pozwala nam gromadzić, generować i eksportować logi, metryki i trace'y aplikacji.
+
+OpenTelemetry umożliwia nam instrumentowanie naszych rozproszonych usług. Instrumentacją nazywamy przechwytywanie danych telemetrycznych ze zdarzeń i operacji w naszym systemie rozproszonym. Jest nam to niezbędne do tego, aby zrozumieć i zbadać zachowanie naszego systemu, znajdować błędy czy debugować problemy związane z wydajnością.
```

Comment:
wywaliłabym wszystkie zbędne zaimki:
OpenTelemetry umożliwia instrumentowanie rozproszonych usług. Instrumentacją nazywamy przechwytywanie danych telemetrycznych ze zdarzeń i operacji w systemie rozproszonym. Jest to niezbędne do tego, aby zrozumieć i zbadać zachowanie systemu, znajdować błędy czy debugować problemy związane z wydajnością.